bad.py

```diff
@@ -0,0 +1,4 @@
+5 == 5
+
+concat_list = ["str1" "str2", "str3"]
```

Comment:
Detected strings implicitly concatenated inside a list.
Python will implicitly concatenate strings when not explicitly delimited.
Was this supposed to be individual elements of the list? ([.common-mistakes.string-concat-in-list.string-concat-in-list](https://semgrep.dev/r/.common-mistakes.string-concat-in-list.string-concat-in-list))

<div>
<a href="https://semgrep.dev">
<img align="right" src="https://r2c.dev/semgrep_icon_text.svg" height=28px alt="" />
</a>

<sub> [How do I suppress this?](https://semgrep.dev/docs/running-rules/#managing-findings) **·** [Report a bug](https://github.com/returntocorp/semgrep-rules/issues/new?assignees=&labels=&template=rule_bug_report.md&title=.common-mistakes.string-concat-in-list.string-concat-in-list) **·** [Manage rule](https://semgrep.dev/manage/projects)</sub>
</div>